pkg/csi/node.go

```diff
@@ -547,9 +548,14 @@ func (n *NodeService) NodeGetInfo(ctx context.Context, _ *csi.NodeGetInfoRequest
 		return nil, fmt.Errorf("failed to get zone for node %s", n.nodeID)
 	}
 
+	nodeMaxVolumeAttachments, err := strconv.ParseInt(node.Labels[NodeLabelMaxVolumeAttachments], 10, 64)
+	if err != nil {
+		nodeMaxVolumeAttachments = DefaultMaxVolumesPerNode
+	}
+
```

Comment:
It might be a good idea to check for negative values and ensure the maximum does not exceed 30; otherwise, we can set it to the default value.

Reply:
or maximum value

Reply:
Will do, only reason I didn't add an upper bound was in case that limit was increased in qemu or proxmox in the future and we don't know how many block devices a user might attach to the node separately. But it definitely makes sense to add. Will update